griptape/config/config.py

```diff
@@ -6,10 +6,21 @@
 from .openai_driver_config import OpenAiDriverConfig
 
 
-@define
+@define(kw_only=True)
 class _Config(BaseConfig):
-    drivers: BaseDriverConfig = field(default=Factory(lambda: OpenAiDriverConfig()), kw_only=True)
-    logging: LoggingConfig = field(default=Factory(lambda: LoggingConfig()), kw_only=True)
+    logging: LoggingConfig = field(default=Factory(lambda: LoggingConfig()))
```

Comment:
Should we be proactive and do this for logging too?

Reply:
I think I'd rather avoid introducing weird hacks like this if we can avoid it. Until there's a clear reason to introduce for logging, I say we avoid -- the API won't change either way.